docs/api/components/legend.en.md

```diff
@@ -3,4 +3,64 @@ title: Legend
 order: 3
 ---
 
-`markdown:docs/api/components/legend.zh.md`
\ No newline at end of file
+`markdown:docs/styles/legend.md`
+
+Legend is an auxiliary element of a chart, which uses color, size and shape to distinguish different data types and is used to filter the data in the chart.
```

Comment:
Legend is an auxiliary component of a plot, which uses color, size, and shape mapping, to show the symbol, color, and name of different series. You can click legends to toggle displaying series in the plot.